setup.py

```diff
@@ -105,6 +105,19 @@ def initialize_options(self) -> None:
     #
     # https://github.com/googleapis/google-auth-library-python/issues/190
     'google-api-python-client>=1.8.0,<1.9.0',
+    #
```

Comment:
This file, one of the 3 largest in the project, increased in size to 357 lines.  The total size of those files is now 3399 lines (target: 1015).  Is this file complex enough to refactor?